variables.tf

```diff
@@ -67,10 +67,11 @@ variable "users" {
 
 variable "vpc_endpoint" {
```

Comment:
We dont have the `endpoint_type` variable while we are using this in the code, would you mind adding this variable and default this to `VPC`? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transfer_server#endpoint_type. This is an oversight from the past I think

Reply:
Hey this is done here: https://github.com/sbpdvb/terraform-aws-mcaf-transfer/blob/feature/add-vpc-endpoint-id-support/main.tf#L55.

Reply:
I see but if you look at variables.tf there is no `variable "vpc_endpoint"` so this does not work

Reply:
The variable was already there: https://github.com/schubergphilis/terraform-aws-mcaf-transfer/blob/master/variables.tf#L68